images/taint-manager/test/clusterrole.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Given we can test this by running the taintmanager locally, are these test files still needed?